weather_mv/loader_pipeline/bq.py

```diff
@@ -52,17 +52,14 @@
 @dataclasses.dataclass
 class ToBigQuery(ToDataSink):
     """Load weather data into Google BigQuery.
-
```

Comment:
Nit: I think we should keep this whitespace.

Reply:
Resolved